### PR TITLE
Release 0.8

### PR DESCRIPTION
**Pull request by SimonHeybrock**

### CODE REVIEW

**Comment by SimonHeybrock** [comment]
Including last-minute improvements to `sc.show()`:

Before
image

After
image

File: docs/environments/scipp-latest.yml [hunk 1]
```diff
@@ -6,6 +6,7 @@ channels:
 
 dependencies:
   - h5py
+  - ipykernel = 6.3.1  # see https://github.com/ipython/ipykernel/issues/771
```
Comment (OwenArnold) [inline docs/environments/scipp-latest.yml:9]: Looks from the issue that we would not need to pin the patch version.

**Review by OwenArnold — APPROVED** [review]
All other changes to documentation look sensible and make use of more modern scipp.

File: python/src/scipp/show.py [hunk 2]
```diff
@@ -106,6 +105,13 @@ def _extents(self):
                 e.append(1)
         return [1] * (3 - len(e)) + e
 
+    def _events_height(self):
+        if self._variable.events is None:
+            return 0
+        events = self._variable.events
+        # Rough estimate of vertical space taken by depiction of events buffer
+        return 1 + 1.3 * (len(events.meta) + len(events.masks))
```
Comment (OwenArnold) [inline python/src/scipp/show.py:113]: Looks like there is a bug here when events is type `Variable`.

**Review by OwenArnold — CHANGES_REQUESTED** [review]

File: python/src/scipp/show.py (hunk 2 above)
Comment (SimonHeybrock) [reply python/src/scipp/show.py:113]: Good catch, thanks.

**Review by OwenArnold — APPROVED** [review]

**Pull request merged**